api/lucy-bot.ts

```diff
@@ -48,7 +45,18 @@ if (!PROD_ENV) {
   bot.launch();
 }
 
-module.exports = (req: NowRequest, resp: NowResponse) => {
+module.exports = async (req: NowRequest, resp: NowResponse): Promise<void> => {
   if (req.method === 'POST') bot.handleUpdate(req.body, resp);
-  else resp.status(200).send('Use POST to use Telegram bot!');
+  else {
+    const replyList = await fetchArticles().then(markdownArticles);
+    await bot.telegram
+      .sendMessage(
+        `${process.env.CHATID}`,
+        `Hey Devs here are week's top blogs :)\n\n${replyList.join('\n\n')}`,
+        <ExtraEditMessage>Extra.markdown().webPreview(false),
+      )
+      .catch((e) => console.error(e));
+
+    resp.status(200).send('Article Sent');
+  }
```

Comment:
Are changes in this file intentional?

Reply:
@nishantwrp so I wasn't able to export the bot properly for bot has to be there to send a message so I used the request, response of lucy-boy.ts which can be changed to something like a query param check to ensure that no one gets to send a message just on pinging lucy's link although I believe link is already hidden

Reply:
Aren't the articles handled by the `article.ts` file.